### PR TITLE
[DOCS] IOS Docs Improvement

### PR DESCRIPTION
**Pull request by mmattel**
Improved documentation.

Went thru every page and fixed/improved user experience.
Resized images, centered images and headlines in tables if necessary, text fixes ect.
Each page changed was tested

### CODE REVIEW

File: docs/modules/ROOT/pages/ios_accounts.adoc [hunk 1]
```diff
@@ -18,81 +24,92 @@ These are:
 
 === Basic Authentication
 
-* Login Credentials are stored on the device (secure in system keychain).
+* Login Credentials are stored on the device (Secured in the iOS system keychain).
```
Comment (EParzefall) [inline docs/modules/ROOT/pages/ios_accounts.adoc:27]: * Login Credentials are stored on the device (Secured in the iOS system keychain).
*

File: docs/modules/ROOT/pages/ios_files.adoc [hunk 2]
```diff
@@ -35,28 +38,29 @@ The actions available for files are:
...
 |===
 
 == Navigate Folders
 
-There are two ways to navigate folders (outside of the root folder).
-To go back to the parent folder, tap btn:[Back] in the top left-hand corner.
-To navigate directly to *any* parent folder tap on the current folder's name.
-When you do, you will see the names of all the parent folders, right up to the root folder, as in the image below.
+There are two ways to navigate folders (outside of the root folder). To go back to the parent folder,
+tap btn:[Back] in the top left-hand corner. To navigate directly to *any* parent folder tap on the
```
Comment (EParzefall) [inline docs/modules/ROOT/pages/ios_files.adoc:59]: *any* parent folder tap on
Add comma after 'folder'.

**Review by EParzefall — COMMENTED** [review]
Looking good. I've made some punctuation suggestions.

File: docs/modules/ROOT/pages/ios_accounts.adoc (hunk 1 above)
Comment (mmattel) [inline docs/modules/ROOT/pages/ios_accounts.adoc:27]:
```suggestion
* Login Credentials are stored on the device (secured in the iOS system keychain).
```

File: docs/modules/ROOT/pages/ios_files.adoc (hunk 2 above)
Comment (mmattel) [inline docs/modules/ROOT/pages/ios_files.adoc:59]:
```suggestion
tap btn:[Back] in the top left-hand corner. To navigate directly to *any* parent folder, tap on the
```

**Comment by michaelstingl** [comment]
@EParzefall changes approved from your side?

**Comment by EParzefall** [comment]
> @EParzefall changes approved from your side?

Yes, definitely, though I wonder what caused the failed checks.

**Comment by mmattel** [comment]
> I wonder what caused the failed checks.

Well known that is has nothing to do with docs... 😄

**Comment by michaelstingl** [comment]
@EParzefall please make sure your approval is shown here:

CleanShot 2021-03-22 at 09 56 26@2x

**Comment by EParzefall** [comment]
> @EParzefall please make sure your approval is shown here:
 
Sorry, @michaelstingl  I don't have the necessary rights for the IOS repo.

File: docs/modules/ROOT/pages/ios_troubleshooting.adoc [hunk 3]
```diff
@@ -34,38 +33,40 @@ Once these have been enabled:
...
 .All iOS App logging settings enabled and set
-image:ios-app-settings-logging.png[ios-app-settings-logging]
+image:ios-app-settings-logging.png[ios-app-settings-logging, width=40%]
 
 === Locating iPhone & iPad App Crash Logs
 
-In the worst case scenario, when the app either isn't responding or is crashing, iOS saves a crash log on the device.
-You can find it under menu:Settings[Privacy > Analytics > Analytics Data].
+In the worst case scenario, when the app either isn't responding or is crashing, iOS saves a crash log
+on the device. You can find it under menu:Settings[Privacy > Analytics > Analytics Data].
```
Comment (hosy) [inline docs/modules/ROOT/pages/ios_troubleshooting.adoc:41]: Why there is a new line after log? Shouldn't this in one line?

**Review by hosy — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/ios_troubleshooting.adoc (hunk 3 above)
Comment (mmattel) [reply docs/modules/ROOT/pages/ios_troubleshooting.adoc:41]: no worry, this renders continuous as one line.
only a special coding at the end of a line forces a linebreak or a blank line between two lines does so.

**Review by hosy — APPROVED** [review]

**Comment by hosy** [comment]
@EParzefall if you can review a pull request you should also be able to approve it. Be sure you selected the right radio button:

Bildschirmfoto 2021-03-22 um 14 10 42

**Comment by hosy** [comment]
@mmattel thank you for updating the documentation. Maybe we should schedule a meeting to go throw all undocumented features.

**Comment by mmattel** [comment]
> Maybe we should schedule a meeting to go throw all undocumented features.

Sure, can you ping me on Talk to clarify the details?

**Review by EParzefall — APPROVED** [review]
Fine improvements.

**Comment by EParzefall** [comment]
@hosy  Duh, had to click on changed files again to get to the review button. Sorry for the confusion.

**Comment by mmattel** [comment]
@hosy 2nd review pls and merge 😄

**Pull request merged**